src/main/java/nextstep/optional/User.java

```diff
@@ -33,7 +35,10 @@ public static boolean ageIsInRange1(User user) {
     }
 
     public static boolean ageIsInRange2(User user) {
-        return false;
+        return Optional.ofNullable(user)
+                .map(User::getAge)
+                .filter(age -> age >= 30 && age <= 45)
+                .isPresent();
```

Comment:
```suggestion
                .filter(age -> age >= 30)
                .filter(age -> age <= 45)
                .isPresent();
```

만약 filter 조건이 복잡하다면 나눠보시는 것도 추천드려요